Adyen/Core/EmptyPaymentComponent.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// Provides a placeholder for payment methods that don't need any payment detail to be filled.
+/// :nodoc:
+public final class EmptyPaymentComponent: PaymentComponent {
```

Comment:
This was on DropIn, right?

Reply:
not any more, its needed in lower level modules